web/beacon-app/src/features/projects/components/TopicTableHeader.tsx

```diff
@@ -25,8 +25,8 @@ const TopicTableHeader: React.FC = () => {
           new services, models, or applications benefit from the data. Then create topics or event
           streams, which are logs that hold messages and events in a logical order. As an event
           broker, Ensign navigates the data for you with speed, ease and accuracy. Need help? See
-          our <Link onClick={handleRedirection}> data flow templates</Link> or{' '}
-          <Link href={PUBLIC_LINKS.OFFICE_HOURS_SCHEDULE} openInNewTab>
+          our <Link href="#"> data flow templates</Link> or{' '}
```

Comment:
Should `#` be used here? Users will see this as a link and click on it, but they will not be redirected anywhere. I think that this shouldn't appear as a link until it has been decided where the users will be redirected.